bundles/binding/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/TellstickBinding.java

```diff
@@ -64,14 +68,20 @@
      */
     private long refreshInterval = 60000;
 
-    private TellstickController controller = new TellstickController();
+    private TellstickController controller;
+    private Thread controllerThread;
+    private SortedMap<TellstickDevice, TellstickSendEvent> messageQue;
 
     public TellstickBinding() {
+        messageQue = Collections.synchronizedSortedMap(new TreeMap<TellstickDevice, TellstickSendEvent>());
+        controller = new TellstickController(messageQue);
+        controllerThread = new Thread(controller);
     }
 
     @Override
     public void activate() {
         logger.info("Activate " + Thread.currentThread());
+        controllerThread.start();

```

Comment:
please do only start any further threads or implementation once you know there are items available for the Tellstick binding (see providesBindingFor() as a possible hook)